src/app-layer-htp.c

```diff
@@ -103,63 +103,60 @@ static uint64_t htp_state_memcnt = 0;
 #endif
 
 SCEnumCharMap http_decoder_event_table[] = {
-    { "UNKNOWN_ERROR", HTTP_DECODER_EVENT_UNKNOWN_ERROR },
-    { "GZIP_DECOMPRESSION_FAILED", HTTP_DECODER_EVENT_GZIP_DECOMPRESSION_FAILED },
-    { "REQUEST_FIELD_MISSING_COLON", HTTP_DECODER_EVENT_REQUEST_FIELD_MISSING_COLON },
-    { "RESPONSE_FIELD_MISSING_COLON", HTTP_DECODER_EVENT_RESPONSE_FIELD_MISSING_COLON },
-    { "INVALID_REQUEST_CHUNK_LEN", HTTP_DECODER_EVENT_INVALID_REQUEST_CHUNK_LEN },
-    { "INVALID_RESPONSE_CHUNK_LEN", HTTP_DECODER_EVENT_INVALID_RESPONSE_CHUNK_LEN },
+    { "UNKNOWN_ERROR", HTP_LOG_CODE_UNKNOWN },
```

Comment:
Is the change from the `HTTP` prefix to `HTP` a change that was made in the libhtp Rust conversion?

Reply:
Indeed, it is a change that vill be made cf https://github.com/OISF/suricata/pull/12508 commit message :

> libhtp logging API updated to provide HTP_LOG_CODE constants along with
    the message. This eliminates the need to perform string matching on
    message text to map log messages to HTTP_DECODER_EVENT values, and the
    HTP_LOG_CODE values can be used directly. In support of this,
    HTP_DECODER_EVENT values are mapped to their corresponding HTP_LOG_CODE
    values:
    
    HTTP_DECODER_EVENT_UNKNOWN_ERROR => HTP_LOG_CODE_UNKNOWN